src/lwc/geFormFieldPicklist/geFormFieldPicklist.js

```diff
@@ -99,6 +103,21 @@ export default class GeFormFieldPicklist extends LightningElement {
         this._recordTypeId = id || this.defaultRecordTypeId;
     }
 
+    get picklistValues() {
+        return isNotEmpty(this.picklistOptionsOverride) ? this.picklistOptionsOverride : this._picklistValues;
+    }
+
+    set picklistValues(value) {
+        this._picklistValues = value;
+    }
+
+    nonePicklistValue() {
```

Comment:
I don't feel too strongly either way here. What do you think about just converting this to a private prop at the top? `_nonePicklistValue = { ... }`